main.go

```diff
@@ -142,7 +161,21 @@ func main() {
 		RHOBSSecretName:              fmt.Sprintf("%v-prom-remote-write", addonName),
 		RHOBSEndpoint:                envVars[rhobsEndpointEnvVarName],
 		RHSSOTokenEndpoint:           envVars[rhssoTokenEndpointEnvVarName],
-	}).SetupWithManager(mgr, nil); err != nil {
+		NetworkType:                  controllers.SDNNetworkType,
+	}
+
+	err = reconciler.Client.Get(ctx, client.ObjectKeyFromObject(egressNetworkPolicyCRD), egressNetworkPolicyCRD)
+
+	if err != nil {
+		if errors.IsNotFound(err) {
+			reconciler.NetworkType = controllers.OVNNetworkType
+		} else {
+			setupLog.Error(err, "Error Getting CRD for EgressNetworkPolicy")
+			os.Exit(1)
+		}
+	}
+
+	if err = (reconciler).SetupWithManager(mgr, nil); err != nil {
```

Comment:
- nit, this method is actually defined on pointer receiver do we need parens around it?
```suggestion
	if err = reconciler.SetupWithManager(mgr, nil); err != nil {
```

Reply:
Let's not mess up with generated code. There is a reason that it was generated that way

Reply:
Done

Reply:
ack